installer/bundle/bundle_skel.sh

```diff
@@ -96,6 +96,7 @@ usage()
     echo "  --force                         Force upgrade (override version checks)."
     echo "  --install                       Install the package from the system."
     echo "  --purge                         Uninstall the package and remove all related data."
+    echo "  --noDigest                      RPM manager skips verification of package or header digests when reading (same as rpm --nodigest --nofiledigest)"
```

Comment:
just wondering do we have an equivalent for DPKG, e.g. should we have this apply as well [--no-debsig](https://manpages.debian.org/testing/dpkg/dpkg.1.en.html#no~4)

Reply:
answered over teams channel :)